interface/client/appStart.js

```diff
@@ -41,6 +39,27 @@ mistInit = function(){
 
 
 Meteor.startup(function(){
+    // send console logging to IPC backend
```

Comment:
Thats exactly what should be as a module in a preloaded file ;)so that not every dapp needs to have that and we can watch all dapps logs, if we wanted to